antarest/study/storage/rawstudy/model/filesystem/matrix/input_series_matrix.py

```diff
@@ -73,14 +75,21 @@ def parse(
                     raise ChildNotFoundError(f"File '{relpath}' not found in the study '{study_id}'") from e
 
             stopwatch.log_elapsed(lambda x: logger.info(f"Matrix parsed in {x}s"))
-            matrix.dropna(how="any", axis=1, inplace=True)
+            matrix = matrix.dropna(how="any", axis=1)
```

Comment:
Why you decided to do this. I think the original way to remove the na is more efficient